qa/tasks/mon_thrash.py

```diff
@@ -212,8 +214,12 @@ def max_killable(self):
 
     def do_thrash(self):
         """
-        Cotinuously loop and thrash the monitors.
+        Continuously loop and thrash the monitors.
         """
+        #status before mon thrashing
+        beforestatus = self.mds_cluster.status()
+        fs = beforestatus.get_filesystems()
```

Comment:
We should check that every file system did not experience a failover.

Reply:
Also, we should only do these checks when enabled for cephfs thrash testing. i.e. make this check a config option to `mon_thrash.py`.

Reply:
@batrick : 

> We should check that every file system did not experience a failover.

Here we are comparing two statuses. Each filesystems are considered inside `hadfailover` function. Are you suggesting to bring that filesystem iteration outside `class FSStatus`? Please clarify.

> make this check a config option to mon_thrash.py
I will make it config option.